packages/selected-network-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [12.0.1]
+
+### Fixed
+
+- When getProviderBlockTracker is called with domain for which there is no cached networkProxy in the domainProxyMap, if the useRequestQueue preference is off and the domain does not have permissions the newly created network proxy for this domain should be pointed at the NetworkController's own proxied selectedNetworkClient. ([#4187](https://github.com/MetaMask/core/pull/4187))
```

Comment:
- The name of the method is `getProviderAndBlockTracker`
- Instead of saying it "should" do something, can we say it will do that thing?

``` suggestion
- When `getProviderAndBlockTracker` is called with a domain for which there is no cached networkProxy in the domainProxyMap, if the useRequestQueue preference is off and the domain does not have permissions, the newly created network proxy for this domain will be pointed at the NetworkController's own proxied selectedNetworkClient. ([#4187](https://github.com/MetaMask/core/pull/4187))

Reply:
done here: https://github.com/MetaMask/core/pull/4188/commits/ecc2c7035cc0467126f35b1d32c56f8689168241

Reply:
done here: https://github.com/MetaMask/core/pull/4188/commits/306bc7148c60040cd4264053d6a0f88a686b6300